modulefiles/modulefile.ProdGSI.orion

```diff
@@ -9,36 +15,37 @@ set COMP_MPI mpiifort
 set C_COMP icc
 set C_COMP_MP mpcc
 
-# Known conflicts
+setenv CRTM_FIX /apps/contrib/NCEPLIBS/orion/fix/crtm_v2.3.0
 
+# Known conflicts
 
-# Load compiler, mpi, cmake, and hdf5/netcdf
-module load intel/2018.4
-module load impi/2018.4
+# Load hpc-stack
+module use /apps/contrib/NCEP/libs/hpc-stack/modulefiles/stack
+module load hpc/1.1.0
+
+# Load intel compiler and mpi
+module load hpc-intel/2018.4
+module unload mkl/2020.2
```

Comment:
May be a note till the bug in hpc-stack is removed.  Pending hpc-stack PR number ...

Reply:
Does the GSI not use MKL? 
If it does, we probably need to load the appropriate mkl module here.

Reply:
Comments explaining mkl/2020.2 added.   Explicitly load mkl/2018.4.